@xen-orchestra/rest-api/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist",
+    "rootDir": "./src",
+
+    "strict": true,
+    "module": "ESNext",
+    "moduleResolution": "bundler",
+    "target": "ESNext",
```

Comment:
Esnext contains a lot of features that may not be implemented, is it possible to target ES2024 ?

Reply:
Looks like `ES2024` is not fully supported on `node 20`.
Is `ES2023` ok for you?